Features/Language.feature

```diff
@@ -0,0 +1,23 @@
+Feature: As a customer I can add edit and delete language 
+
+
+Scenario: Add new Language
+
+Given I navigate to Language
+When I add a new langauge English with level Fluent
+Then I expect add success message
+
+
+
+Scenario: Edit Language
+
+Given I navigate to Language 
+When I edit language to German with level Conversational
+Then I expect update success message
+
+
+Scenario: Delete Language
+
+Given I navigate to Language
+When I delete an existing language German
```

Comment:
Please use tables  and pass values to your scenarios for example look at the below scenario 
This way you will get exposure to use them  and will know how to pass test data to the tests


Example 

	Given I logged into the portal
	When I click on tab Education
	And I add my education 
	Then I am able to see my education'

Would  change to 


@Education
Scenario Outline: 1-Add education
	Given I logged into the portal
	When I click on tab Education
	And I add my education including '<Country>', '<University>', '<Title>', '<Degree>', '<Graduation Year>'
	Then I am able to see my education details including '<Country>', '<University>', '<Title>', '<Degree>', '<Graduation Year>'

	Examples:
	| Country     | University                      | Title  | Degree  | Graduation Year |
	| New Zealand | Whitireia Community Polytechnic | M.Tech | Diploma | 2021            |
	| Venezuela   | Hhutech University              | B.A    | Diploma | 2011            |

See  https://cucumber.io/docs/gherkin/reference/

Reply:
As suggested, I have used tables and passed values from it (please refer below)
https://github.com/Godzeye99/Mars_QA_repo/blob/onboarding_task/Features/Language.feature#L13
https://github.com/Godzeye99/Mars_QA_repo/blob/onboarding_task/Features/Skill.feature#L13